README.md

```diff
@@ -1,41 +1,41 @@
-# CoursesSite
+# Courses
 
 <a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
 
 ✨ **This workspace has been generated by [Nx, Smart Monorepos · Fast CI.](https://nx.dev)** ✨
 
-## Integrate with editors
+## Setup
 
-Enhance your Nx experience by installing [Nx Console](https://nx.dev/nx-console) for your favorite editor. Nx Console
-provides an interactive UI to view your projects, run tasks, generate code, and more! Available for VSCode, IntelliJ and
-comes with a LSP for Vim users.
+This repo uses [pnpm](https://pnpm.io/) as the package manager. To install dependencies, run `pnpm install`.
+
+To install Nx globally, run `pnpm add --global nx@latest`.
 
 ## Start the application
 
-Run `npx nx dev courses-site` to start the development server. Happy coding!
+Run `nx dev courses-site` to start the development server. Happy coding!
 
 ## Build for production
 
-Run `npx nx build courses-site` to build the application. The build artifacts are stored in the output directory (e.g. `dist/` or `build/`), ready to be deployed.
+Run `nx build courses-site` to build the application. The build artifacts are stored in the output directory (e.g. `dist/` or `build/`), ready to be deployed.
```

Comment:
I personally prefer to use `npx` as I find globally installing NX to make it work a smell

Reply:
yes I had thought about this but just preferred to type nx as easier on the keyboard 😛 I'll switch it back, I don't know why nx suggests to install globally seems odd way to ensure correct versions.